src/main/java/core/storage/FruitOperations.java

```diff
@@ -0,0 +1,10 @@
+package core.storage;
+
+public interface FruitOperations {
+
+    public boolean supply(Fruit fruit, int quantity);
+
+    public boolean buy(Fruit fruit, int quantity);
+
+    public boolean refund(Fruit fruit, int quantity);
+}
```

Comment:
let's use the strategy pattern:
you have the `FruitOperation` interface, let's have only one `apply()` method here

create three classes implementing this interface: `SupplyOperation`, `BuyOperation` and `ReturnOperation`: their `apply()` method will call the necessary action related to the storage

then, after you have parsed the lines of your csv file and made a `FruitPackageDto` instance out of each line - check the `orderType` field of your `FruitPackageDto` instance and based on its value choose the necessary `FruitOperation` implementation